api/v1/latency.go

```diff
@@ -0,0 +1,71 @@
+package api
+
+import (
+	"encoding/json"
+	"net/http"
+	"time"
+
+	"github.com/celestiaorg/bittwister/xdp/latency"
+	"go.uber.org/zap"
+)
+
+// PacketlossStart implements POST /packetloss/start
+func (a *RESTApiV1) LatencyStart(resp http.ResponseWriter, req *http.Request) {
+	var body LatencyStartRequest
+	if err := json.NewDecoder(req.Body).Decode(&body); err != nil {
+		sendJSONError(resp,
+			MetaMessage{
+				Type:    APIMetaMessageTypeError,
+				Slug:    SlugJSONDecodeFailed,
+				Title:   "JSON decode failed",
+				Message: err.Error(),
+			},
+			http.StatusBadRequest)
+		return
+	}
+
+	if a.lt == nil {
+		a.lt = &netRestrictService{
+			service: &latency.Latency{
```

Comment:
[question] Is setting latency and jitter at the same endpoint required, or can that be splitted into separate endpoints?